Resources/Prototypes/Entities/Objects/Weapons/Melee/caveman_club.yml

```diff
@@ -0,0 +1,30 @@
+- type: entity
+  name: caveman club
+  parent: [BaseItem, BaseMinorContraband]
+  id: CavemanClub
+  description: It brings forth primal urges, OOK!
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/caveman_club.rsi
+    state: icon
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    damage:
+      types:
+        Blunt: 10
+    soundHit:
+      collection: MetalThud
+  - type: Item
+    size: Normal
+  - type: Clothing
+    quickEquip: false
+    slots:
+    - back
+  - type: Construction
+    graph: CavemanClub
+    node: club
+  - type: UseDelay
+    delay: 1
```

Comment:
```suggestion
```
Not needed since the attack rate is defined in MeleeWeaponComponent. The baseball bat has this for wielding I think.

Reply:
whoopsie youre right